tutorials/deploy-penpot-with-docker-instantapp/index.mdx

```diff
@@ -0,0 +1,163 @@
+---
+meta:
+  title: How to deploy a self-hosted Penpot Instance using the Docker InstantApp
+  description: This page shows you how to deploy a self-hosted Penpot Instance using the Docker InstantApp
+content:
+  h1: How to ddeploy a self-hosted Penpot Instance using the Docker InstantApp
+  paragraph: This page shows you how to deploy a self-hosted Penpot Instance using the Docker InstantApp
+tags: penpot docker self-hosted
+categories:
+  - compute
+dates:
+  validation: 2022-09-28
+  posted: 2022-09-28
+---
+
+Penpot is the an open-source design and prototyping platform meant for cross-domain teams. It is completely web based and works with open web standards (SVG). Penpot can run in almost any modern browser, no matter of the underlying operating system.
+
+In this tutorial you will learn how to deploy your own Penpot Instance using the Docker [InstantApp](https://www.scaleway.com/en/docs/tutorials/docker-instantapp/).
+
+## Installing Penpot using Docker Compose 
+
+1. In the Scaleway console click **Instances** in the **Compute** section of the navigation bar. Then [deploy](https://console.scaleway.com/instance/servers/create?area=fr-par-1&imageKey=c1b530d8-0ca0-45c4-80db-ba06608287b2&offerName=PLAY2-MICRO) a new Instance using the Docker InstantApp].
```

Comment:
Il y a un crochet en trop après InstantApp, or did you want to put a link maybe?

Reply:
```suggestion
1. In the Scaleway console click **Instances** in the **Compute** section of the navigation bar. Then [deploy](https://console.scaleway.com/instance/servers/create?area=fr-par-1&imageKey=c1b530d8-0ca0-45c4-80db-ba06608287b2&offerName=PLAY2-MICRO) a new Instance using the Docker InstantApp.
```